db/models.py

```diff
@@ -1 +1,23 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True, null=True)
```

Comment:
The `name` field in the `Guild` model is marked as `unique=True` and `null=True`. This combination can lead to multiple `null` entries, which violates the uniqueness constraint. Consider removing `null=True` to enforce uniqueness properly.